src/extension.ts

```diff
@@ -33,24 +41,49 @@ class PSCompletionItem extends CompletionItem {
 }
 
 class PSCompletionProvider implements CompletionItemProvider {
-  constructor(private snippets: [Snippet]) {}
+  constructor(private snippets: Snippets) {}
 
   provideCompletionItems(): ProviderResult<CompletionItem[]> {
     return Object.keys(snippets).map(key => new PSCompletionItem(snippets[key]))
   }
 }
 
 export async function activate(context: ExtensionContext) {
-  let registerCompletionProvider = languages.registerCompletionItemProvider(
-    'javascript',
-    new PSCompletionProvider(snippets as [Snippet]),
-    '*'
-  )
+  async function workspaceFoldersChange(folders: WorkspaceFolder[]) {
+    let config
+    if (folders) {
+      config = await prettier.resolveConfig(folders[0].uri.path)
```

Comment:
I'm convinced that `fsPath` is what's neded here.

– [link](https://code.visualstudio.com/docs/extensionAPI/vscode-api#_a-nameuriaspan-classcodeitem-id373urispan)